devito/types/basic.py

```diff
@@ -211,6 +211,14 @@ def _mem_shared(self):
         """
         return False
 
+    @property
+    def _mem_shared_remote(self):
```

Comment:
I assume this is for grace-hoper where the GPU can access the grace's shared memory?

Reply:
no, this is for just hopper where a block X running on SM0 can access the shared memory of block Y on the neighbouring SM1